src/public/views/changelist.njk

```diff
@@ -20,12 +20,29 @@
   <div class="row">
     <div class="col-12">
       <table class="table table-bordered table-striped entity-table">
+        <tr>
```

Comment:
this row should be in a `thead`, and can you make it dark for extra street cred? https://getbootstrap.com/docs/4.3/content/tables/#table-head-options